specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -10391,7 +10391,19 @@
           ],
           "x-ms-enum": {
             "name": "OSType",
-            "modelAsString": false
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "linux",
+                "description": "The Linux operating system.",
+                "name": "linux"
```

Comment:
Don't need specify "name" if not override the enum name.

Reply:
I will fix this